pkg/server/container_status.go

```diff
@@ -70,6 +74,20 @@ func toCRIContainerStatus(container containerstore.Container, spec *runtime.Imag
 			reason = errorExitReason
 		}
 	}
+	if verbose {
+		info = make(map[string]string)
+		m, _ := json.Marshal(status.Pid)
+		info["pid"] = string(m)
+		m, _ = json.Marshal(meta.Config)
+		info["container config"] = string(m)
+		m, _ = json.Marshal(status.Removing)
+		info["removing state"] = string(m)
```

Comment:
Not needed, I think. :)

Reply:
Yeah but it's for "our" purposes in case it gets messed up.. :-) I thought you might not want it. Think it over a bit then I'll remove if you are still against it.